mandatory/1-fix-functions.js

```diff
@@ -9,21 +9,18 @@
   NOTE:Only make edits inside the function
 
 */
-
-function getMood() {
-  let isHappy = true;
-
+let isHappy = false;
```

Comment:
hey!
so what you've done here is define a variable called 'isHappy' and then below you've used it as a function parameter. This is called [Variable Shadowing](https://www.geeksforgeeks.org/variable-shadowing-in-javascript/) - it technically works but i can be confusing and unnecessary, because it means that the first use of `isHappy` doesn't mean anything. 
You don't need to define isHappy as a variable before the function - defining it as a parameter in the function means it is sort of like a placeholder for when you pass it in when calling the function later!